django/contrib/admin/templates/admin/app_index.html

```diff
@@ -4,14 +4,16 @@
 {% block bodyclass %}{{ block.super }} app-{{ app_label }}{% endblock %}
 
 {% if not is_popup %}
-{% block breadcrumbs %}
-<div class="breadcrumbs">
-<a href="{% url 'admin:index' %}">{% translate 'Home' %}</a>
-&rsaquo;
-{% for app in app_list %}
-{{ app.name }}
-{% endfor %}
-</div>
+{% block nav-breadcrumbs %}
+  <nav aria-label="{% translate 'Breadcrumbs' %}">
+    <div class="breadcrumbs">
+      <a href="{% url 'admin:index' %}">{% translate 'Home' %}</a>
+      &rsaquo;
+      {% for app in app_list %}
+        {{ app.name }}
+      {% endfor %}
+    </div>
+  </nav>
```

Comment:
Nice 👍 A little more "proper" than what proposed in the ticket. There's some duplication but it's minor.

Sometimes in codebases I like to put in a "if you edit this please also edit this other thing" comment with WET code across files like this... not necessary but just a suggestion 😁

Reply:
We have a test that covers this duplication, so it should be fine.